#include "testes.hpp"

void Testes::CodigoApresentacao_SetComValorErrado(){ // throw(invalid_argument);

    throw invalid_argument("CodigoApresentacao_SetComValorErrado: Terminado com falha.");

}